build-support/bin/release.sh

```diff
@@ -298,17 +245,14 @@ function install_and_test_packages() {
   export PANTS_PLUGIN_CACHE_DIR=$(mktemp -d -t plugins_cache.XXXXX)
   trap "rm -rf ${PANTS_PLUGIN_CACHE_DIR}" EXIT
 
-  PACKAGES=("${RELEASE_PACKAGES[@]}")
+  packages=($(run_local_pants -q run src/python/pants/releases:packages -- list | grep '.' | awk '{print $1}'))
 
   export PANTS_PYTHON_REPOS_REPOS="${DEPLOY_PANTS_WHEEL_DIR}/${VERSION}"
-  for PACKAGE in "${PACKAGES[@]}"
+  for package in "${packages[@]}"
   do
-    NAME=$(pkg_name $PACKAGE)
-    INSTALL_TEST_FUNC=$(pkg_install_test_func $PACKAGE)
-
-    start_travis_section "${NAME}" "Installing and testing package ${NAME}-${VERSION}"
-    eval $INSTALL_TEST_FUNC ${PIP_ARGS[@]} || \
-      die "Failed to install and test package ${NAME}-${VERSION}!"
+    start_travis_section "${package}" "Installing and testing package ${package}-${VERSION}"
+    eval pkg_${package##*\.}_install_test ${PIP_ARGS[@]} || \
```

Comment:
Note: I renamed a couple of the `pkg_..._install_test` functions to all adhere to this pattern. They are currently all unique.